docs/guides/modules/orchestrate/pages/set-up-triggers.adoc

```diff
@@ -0,0 +1,121 @@
+= Set up triggers
+:page-platform: Cloud, Server v4+
+:page-description: A guide to setting up triggers for your pipelines.
+:experimental:
+
+Triggers are configurable rules that determine when to automatically run a pipeline based on specific events and conditions.
+
+A project can have one or more triggers, which are associated with events from a source of change. The most common type of trigger is a VCS event, for example, a push to a branch on a GitHub repository. Any system that can emit outbound webhooks can be set up as a trigger. A trigger can have only one pipeline but a pipeline can have multiple triggers.
+
+A trigger kicks off a pipeline to run the workflows defined in your connected config source.
+
+== Add a trigger
+
+NOTE: Bitbucket Cloud triggers are created automatically when setting up a project and they cannot be edited. The only additional trigger types available for Bitbucket Cloud pipelines are xref:scheduled-pipelines.adoc[scheduled triggers].
+
+NOTE: **Using CircleCI server?** The Pipeline and Trigger pages under Project Settings and pipeline and trigger add/edit functionality is **not** available. CircleCI server pipelines are set up when a project is set up and cannot be edited. CircleCI server requires CircleCI configuration files to be stored in the same repository as your code at `.circleci/config.yml`.
+
+The pipeline and trigger options and features available to you depend on how your code is integrated and where it is stored. For a feature availability overview, see the xref:integration:version-control-system-integration-overview.adoc[VCS integration overview] page.
+
+There are broadly three types of triggers:
+
+* VCS triggers.
+* Custom webhooks used for triggering pipelines from external services.
+* Schedule triggers.
+
+To add a trigger for a pipeline, follow these steps:
+
+[tabs]
```

Comment:
Technically this is missing the GitHub OAuth trigger

Reply:
I wonder if a table like this would be helpful here.

<img width="450" height="666" alt="Screenshot 2025-09-19 at 15 09 30" src="https://github.com/user-attachments/assets/9b291317-1b45-4835-aa09-5adc5d9d01f5" />